ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
```

Comment:
What is the reason that we are targeting a dummy repo in CSPI-QE here?

I'd imagine that this testing would be helpful to someone working on rhods. Can we target a repo that is actively being work on by devs/qe in the rhods space?

Reply:
The purpose of the team is to run integration tests; we are tied to openshift ci requirement to have a repo even if we do not need one.
We are not going to be tied to a qe team repo; qe teams do not have to work in github.
The config is relevant to the team. Reuse and useful actions should be in step registry.